lib/location/location_search_storage.dart

```diff
@@ -23,9 +23,13 @@ class LocationSearchStorage {
   void load(BuildContext context, String key) async {
     _places.clear();
     _streets.clear();
-    final locationData = await loadFromAssets(context, key);
-    _places.addAll(locationData.places);
-    _streets.addAll(locationData.streets);
+    try {
+      final locationData = await loadFromAssets(context, key);
+      _places.addAll(locationData.places);
+      _streets.addAll(locationData.streets);
+    } on FlutterError catch (e) {
+      print(e);
```

Comment:
I have not yet a better solution to test this, if you have better ideas feel free to give me a hint :)